api/src/main/java/uk/gov/dhsc/htbhf/claimant/message/context/MessageContextLoader.java

```diff
@@ -73,11 +73,11 @@ public MakePaymentMessageContext loadMakePaymentContext(Message message) {
      * @param message The message to inflate.
      * @return A wrapper object with the inflated objects
      */
-    public NewCardMessageContext loadNewCardContext(Message message) {
-        NewCardRequestMessagePayload payload = payloadMapper.getPayload(message, NewCardRequestMessagePayload.class);
+    public RequestNewCardMessageContext loadNewCardContext(Message message) {
```

Comment:
This method should probably be renamed to `loadRequestNewCardContext` otherwise it will clash when the new message is created which will need a similar method like `loadCreateNewCardContext`.

Reply:
Renamed.